crafters/nlp/PDFExtractorSegmenter/tests/test_pdfextractorcrafter.py

```diff
@@ -12,97 +12,69 @@
 path_img = os.path.join(cur_dir, 'cats_are_awesome_img.pdf')
 
 
-def test_io_uri_images_and_text():
-    crafter = PDFExtractorSegmenter()
-    chunks = crafter.craft(uri=path_img_text, buffer=None)
+def test_images_and_text():
+    for i in range(2): #Test for URI and buffer
+        if i is 0: #Data coming from URI
```

Comment:
not sure in Python `i is 0` would work since is does check the object and not the value, what you want here is `if i == 0`.

But anyway do split the test